### DISCUSSION TITLE
How do I add french to the "Use do-not-break-after list" ?

### DISCUSSION THREAD

**Discussion by devkatalist** [SubtitleEdit/subtitleedit#7745] [General]
How do I add french to the "Use do-not-break-after list" ?

**Comment by niksedk** [comment]
We could add a default `fr_NoBreakAfterList.xml` (in the Dictionaries folder) file in SE... what could a minimal list look like?

````xml
<NoBreakAfterList>
  <Item>Dr</Item>
  <Item>Dr.</Item>
  <Item>Mr.</Item>
  <Item>Mrs.</Item>
  <Item>Ms.</Item>
</NoBreakAfterList>
````

**Reply by devkatalist** [reply]
In this case, it would work only for french language, right? What if a user of another language wants to add their own language? 
Could there be a button on SE to add language about here:

![button](https://github.com/SubtitleEdit/subtitleedit/assets/37704802/6ee54289-9793-40cc-a8b1-fdae9f473e87)

**Comment by niksedk** [comment]
OK, how is this?
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/6a80c4ef-c7f7-48ff-9e70-1f39aaa380b0)


https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.2/SubtitleEditBeta.zip

A minimal French list would still be appreciated.

**Comment by devkatalist** [comment]
That would be perfect! And as you said, a minimal French list is also a viable option.

**Reply by niksedk** [reply]
It's included in latest beta...

**Comment by devkatalist** [comment]
Thank you so much! It's really appreciated!
